teams/wg-cli.toml

```diff
@@ -2,18 +2,19 @@ name = "wg-cli"
 kind = "working-group"
 
 [people]
-leads = ["spacekookie"]
```

Comment:
Would love to get @spacekookie's confirmation on this just to make sure everyone is on the same page.